src/components/DataActionButton/index.tsx

```diff
@@ -24,7 +24,13 @@ type DataActionButtonProps = {
 };
 
 function DataActionButton(props: DataActionButtonProps) {
-  const { pathOrKey } = props;
+  const { bucketOrVolume, pathOrKey, type } = props;
+  let uri = '';
+  if (type == 's3') {
+    uri = `s3://${bucketOrVolume}/${pathOrKey}`;
+  } else if (type == 'gds') {
+    uri = `gds://${bucketOrVolume}/${pathOrKey}`;
```

Comment:
If GDS, it wouldn't need slash in between. GDS path starts with a forward slash `/`.

```
uri = `gds://${bucketOrVolume}${pathOrKey}`;
```

Reply:
Ahh, yes thanks for pointing that out!